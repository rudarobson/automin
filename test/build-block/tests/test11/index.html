<!DOCTYPE html>
<html>

<body>
	<!-- build:sass index.css {
		"absolute":true
	}-->
	<link rel="stylesheet" type="text/css" href="index.scss" />
	<!-- /build -->
</body>

</html>
