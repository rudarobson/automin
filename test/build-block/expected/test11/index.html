<!DOCTYPE html>
<html>

<body>
	<link rel="stylesheet" type="text/css" href="/tmp/build-block/index.css" />
</body>

</html>
